src/routing/RouteSelector.jsx

```diff
@@ -43,9 +42,10 @@ export const RouteSelector = () => (
     <ProtectedRouteWithProps exact path="/sites/:siteName/folder/:folderName/subfolder/:subfolderName" component={Folders} />
     <ProtectedRouteWithProps exact path="/sites/:siteName/navbar" component={EditNavBar} />
     <ProtectedRouteWithProps path="/sites/:siteName/files/:fileName" component={EditFile} />
-    <ProtectedRouteWithProps path="/sites/:siteName/files" component={Files} />
-    <ProtectedRouteWithProps path="/sites/:siteName/images/:customPath" component={Images} />
-    <ProtectedRouteWithProps path="/sites/:siteName/images" component={Images} />
+    <ProtectedRouteWithProps path="/sites/:siteName/documents/:customPath" component={Media} mediaType={'documents'} />
+    <ProtectedRouteWithProps path="/sites/:siteName/documents" component={Media} mediaType={'documents'} />
+    <ProtectedRouteWithProps path="/sites/:siteName/images/:customPath" component={Media} mediaType={'images'} />
+    <ProtectedRouteWithProps path="/sites/:siteName/images" component={Media} mediaType={'images'} />
```

Comment:
As a suggestion, what do you think of locating the `images` and `documents` root folder routes as `"/sites/:siteName/documents/home"` and `"/sites/:siteName/images/home"`, which would allow us to simplify the `ProtectedRouteWithProps` for images and documents from 4 to 2

Reply:
As discussed offline, using `/sites/:siteName/documents/home` will prevent users from having a media subfolder named `home`. We could work around this by prefixing all media folder routes with `home`, but we have elected to leave it as is for now for greater clarity.